packages/core/src/components/field-label/field-label.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import { color } from '@onfido/castor';
+import { html, Meta, omit, Story } from '../../../../../docs';
+import { Asterisk } from '../asterisk/asterisk.story';
+import { Field } from '../field/field.story';
+import { HelperText } from '../helper-text/helper-text.story';
+import { Input } from '../input/input.story';
+import { Textarea } from '../textarea/textarea.story';
+import { FieldLabel, FieldLabelProps } from './field-label.story';
+
+export default {
+  title: 'Core/FieldLabel',
+  component: FieldLabel,
+  argTypes: omit<FieldLabelProps>('for'),
+  args: {
+    children: 'Label',
+  },
+} as Meta<FieldLabelProps>;
+
+export const Playground: Story<FieldLabelProps> = (props) => FieldLabel(props);
+
+export const AsOptional: Story<FieldLabelProps> = ({ children, ...props }) =>
+  FieldLabel({
+    ...props,
+    children: html('span', {
+      children: [
+        children,
+        html('span', {
+          style: `color: ${color('content-secondary')}`,
+          children: '(optional)',
+        }),
+      ],
+    }),
+  });
+
+export const AsRequired: Story<FieldLabelProps> = ({ children, ...props }) =>
+  FieldLabel({
+    ...props,
+    children: html('span', {
+      children: [children, Asterisk({ 'aria-label': 'required' })],
+    }),
+  });
+
+interface FieldLabelWithHelperTextProps extends FieldLabelProps {
+  label: string;
+  helperText: string;
+}
+
+export const WithHelperText: Story<FieldLabelWithHelperTextProps> = ({
+  label,
+  helperText,
+  ...props
+}) =>
+  FieldLabel({
+    ...props,
+    children: [label, HelperText({ children: helperText })],
+  });
+WithHelperText.argTypes = omit<FieldLabelWithHelperTextProps>('children');
+WithHelperText.args = {
+  label: 'Label',
+  helperText: 'Helper text',
+};
+
+interface FieldLabelWithInputProps extends FieldLabelProps {
+  id: string;
+  label: string;
+}
+
+export const WithInput = ({ id, label, ...props }: FieldLabelWithInputProps) =>
+  Field({
+    children: [
+      FieldLabel({
+        ...props,
+        children: label,
+        for: id,
+      }),
+      Input({ id }),
+    ],
+  });
```

Comment:
Remind me again why we do `input` outside of `label` with `[id]` and `[for]`?

Reply:
i.e. 
![image](https://user-images.githubusercontent.com/18623773/112667637-6b63d080-8e55-11eb-82b9-5bc6a005a5a2.png)

Reply:
This was to demonstrate different ways of using field label with an input.

FieldLabel has examples of using `[for]`, where Input examples show as using nesting within `label`.

However, I have noticed that on HTML + CSS I used `[for]` everywhere... Will change.